media-video/amdgpu-pro-amf/amdgpu-pro-amf-1.4.31.1654522.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit unpacker
+
+MY_PV_HIGH=$(ver_cut 1)
+MY_PV_MIDDLE=$(ver_cut 2)
+MY_PV_LOW=$(ver_cut 3)
+MY_PV_REV=$(ver_cut 4)
+
+MY_PV="${MY_PV_HIGH}.${MY_PV_MIDDLE}.${MY_PV_LOW}"
+MY_PV_FULL="${MY_PV}-${MY_PV_REV}"
+
+MY_PN="amf-amdgpu-pro"
+
+INTERNAL_VER="5.7"
+UBUNTU_VER="22.04"
+
+DESCRIPTION="AMD's closed source Advanced Media Framework (AMF) driver"
+HOMEPAGE="https://www.amd.com/en/support"
+SRC_URI="https://repo.radeon.com/amdgpu/${INTERNAL_VER}/ubuntu/pool/proprietary/a/${MY_PN}/${MY_PN}_${MY_PV_FULL}.${UBUNTU_VER}_amd64.deb -> ${P}.deb"
+
+S="${WORKDIR}"
+
+RESTRICT="bindist mirror"
+
+LICENSE="AMD-GPU-PRO-EULA"
+SLOT="0"
```

Comment:
```
CMP: media-video/amdgpu-pro-amf-1.4.30.1620044/image with media-video/amdgpu-pro-amf-1.4.31.1654522/image
 FILES:-usr/lib64/libamfrt64.so.1.4.30
 FILES:+usr/lib64/libamfrt64.so.1.4.31
   ABI: libamfrt64.so.1(64) func(+47,-14) vars(+6,-2) [BREAKING]
------> FILES(+1,-1) ABI(+53,-16,>B<)
```
Aww crap.

Reply:
I should be binding `amdgpu-pro-amf` with their corresponding vulkan drivers ?

Reply:
Sorry I don't understand the question. But this led me to do 
89c6f54d2280ce974f1dfa47389ec87f458a0176
e887ce7bd6dcb082de177e506935af71212c2912

Note that in future if the update introduces breaking SONAME changes, you'll have to bump the subslot value to match the SONAME value.

Reply:
Sorry, basically `amdgpu-pro-amf` needs `amdgpu-pro-vulkan` to work, I wasn't sure how to interpret the code snippet you attached, looked like an API or ABI change ? so probably `amdgpu-pro-amf` needs to depend on a specific version of `amdgpu-pro-vulkan` ? 

ffmpeg depends only on `amf-headers` when it gets built, why would it need to be rebuilt with changing versions of `amdgpu-pro-amf` (binary only) ?